src/libraries/libraries-packages.proj

```diff
@@ -18,6 +18,7 @@
     <ProjectReference Include="$(PkgDir)*\*.proj" Exclude="$(PkgDir)test\*" />
     <ProjectReference Include="$(MSBuildThisFileDirectory)*\pkg\**\*.pkgproj" Condition="('$(BuildAllConfigurations)' == 'true' or '$(DotNetBuildFromSource)' == 'true') And '$(BuildAllOOBPackages)' == 'true'" />
     <!-- If setting BuildAllOOBPackages to false, add bellow the individual OOB packages you want to continue to build -->
```

Comment:
Where is `BuildAllOOBPackages` set?

Reply:
A few lines above this one on this same file:

https://github.com/dotnet/runtimelab/blob/82c3b4c7fa34091524d21fe4ed8977c2da893570/src/libraries/libraries-packages.proj#L7